content/docs/start/data-pipelines.md

```diff
@@ -1,13 +1,24 @@
-# Data Pipelines
+---
+title: 'Get Started: Data Pipelines'
+description: 'Get started with pipelines in DVC. Learn how to capture, organize,
```

Comment:
Is it desirable to repeat the title more or less in the description? Curious

Reply:
Whatever is best to appeal to the audience. It isn't used for indexing purposes, but including terms that match popular searches (which might then get highlighted) can be helpful to that end. The key objective is increasing clickthrough and reducing bounce, so a copy/technical writing mindset works well.

In this case, a lot of the searches are quite general: "machine learning pipelines", "ml pipelines". So I wrote the description to appeal to people who know they want a pipeline, but might not have heard of DVC or built a pipeline before. (Sometimes challenging in < 160 characters, haha)

Reply:
Right, but my question is more basic than that: if the search result title is "Get Started: Data Pipelines" , isn't starting the description with "Get started with pipelines..." a waste of real estate (since we only have 160 chars)? Seems kind of redundant to me (it's not like people don't read the title but only read the desc in a serach result).

Reply:
OK, edited it to avoid repetition.

Reply:
Ty